src/components/PageViewPr.js

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { Button } from 'react-bootstrap'
+import { uiViewPrGetData, uiViewPrGetBaseUrl, uiViewPrUpdateList } from '../actions/ui'
+import { ROUTE_VIEW_PR } from '../utils/routingConstants'
+import { uiNavigation } from '../actions/ui'
+import AbstractPageDefinitions from './AbstractPageDefinitions'
+
+class PageViewPr extends AbstractPageDefinitions {
```

Comment:
This ought to be `PageContribution` and the URL should be `clearlydefined.io/contributions/174`

"view pr" is too implementation specific.

This naming change will affect a lot of naming. like the action names, the route name and value, ...